neptune/new/internal/backends/hosted_neptune_backend.py

```diff
@@ -281,7 +281,7 @@ def get_metadata_container(
                 .result
             )
 
-            if experiment.type != container_type:
+            if experiment.type != container_type.value:
```

Comment:
You could actually change `ContainerType` class from `class ContainerType(enum.Enum)` to `class ContainerType(str, enum.Enum)` and solve such problems once and for all. Makes sense?